monitoring/types.go

```diff
@@ -23,6 +24,7 @@ const (
 	instanceManagerLabel = "instance_manager"
 	instanceManagerType  = "instance_manager_type"
 	managerLabel         = "manager"
+	backupLabel          = "backups"
```

Comment:
Change the value from backups to backup.

Reply:
Fixed in b92e5cc21e2211ccc124ef4234b3288a8d413e26